modules/new_profile/test/TestPlan.md

```diff
@@ -0,0 +1,27 @@
+# New Profile Test Plan:
+
+1. Access New Profile page and ensure that it renders.
+   [Automation Testing]
+2. Access with useEDC on/off and ensure that the field appears/disappears
+   [Manual Testing]
```

Comment:
@ridz1208 Not a blocker but could you ensure that all phrases either end with a dot (preferably) or do not end with a dot?

Reply:
@PapillonMcGill ma bad. Fixed it